scripts/validate

```diff
@@ -11,5 +11,13 @@ fi
 echo Running: golangci-lint
 golangci-lint run
 
+echo Tidying up modules
+go mod tidy
+
 echo Verifying modules
 go mod verify
+
+if [[ $(git diff --stat) != '' ]]; then
```

Comment:
Any reason we used `git diff` ? Typically we use [`git status --porcelain`](https://github.com/search?q=org%3Arancher+porcelain&type=code). Not a big difference in functionality, but could be parsing subtleties.

Reply:
I don't have a preference, ill switch it to porcelain to make it consistent.